File: app/src/main/java/com/eericxu/wallpaper/service/Wallpaper.java
package com.eericxu.wallpaper.service;


/**
 * Created by Eericxu on 2017-06-05.
 */

public interface Wallpaper {
    public static final int Runing=1;
    int state=0;
    void onCreate(WallpaperServer.MEngine engine);
    void destroy();

    void onVisibility(boolean visible);
}

File: app/src/main/java/com/eericxu/wallpaper/service/Wallpaper.kt
package com.eericxu.wallpaper.service

import com.eericxu.wallpaper.service.WallpaperServer.MEngine


/**
 * Created by Eericxu on 2017-06-05.
 */
interface Wallpaper {
    fun onCreate(engine: MEngine)
    fun destroy()

    fun onVisibility(visible: Boolean)
}
